### PR TITLE
Newsletters: add 68 (2019-10-16)

### PR DESCRIPTION
**Pull request by harding**
- [x] Check for updated RCs
- [x] Check for YouTube videos made public

### CODE REVIEW

File: _posts/en/newsletters/2019-10-16-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+  default in the upcoming 0.19.0 release.  This change to remove them is
+  part of the master development branch and is expected to be released
+  in 0.20.0, more than a year after the initial announcement of
+  deprecation covered in [Newsletter #13][pr14054] and followed up in
+  Newsletters [#37][bip61 discussion] and [#38][bip61 decision].
+
+- [Bitcoin Core #17056][] adds a `sortedmulti` [output script
+  descriptor][] that sorts the pubkeys provided to it using the
+  lexicographic order described in [BIP67][].  This makes it possible to
+  use import xpub-based descriptors for wallets that require using BIP67,
```
Comment (jnewbery) [inline _posts/en/newsletters/2019-10-16-newsletter.md]: s/makes it possible to use import/makes it possible to import/

**Comment by jnewbery** [comment]
ACK ea04b57d3e983bc552f09a673a1b14d29668b7e1

Just need to check latest RCs before merge.

**Comment by jnewbery** [comment]
Does the newsletter summary also need to be changed now that LND is a release, rather than a release candidate? It currently reads:

> This week's newsletter requests testing of several release candidates, relays an update on the taproot proposal, describes a proposed increase...

Perhaps change to:

> This week's newsletter announces the newest release of LND, requests testing of release candidates for Bitcoin Core and C-Lightning, relays an update on the taproot proposal, describes a proposed increase...

File: _posts/en/newsletters/2019-10-16-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+  so that any problems can be found and fixed before their general
+  public release:
+
+    - [Bitcoin Core 0.19.0rc1][bitcoin core 0.19.0]
+    - [C-Lightning 0.7.3-rc2][c-lightning 0.7.3]
+
+## News
+
+- **Taproot update:** Pieter Wuille sent an [email][taproot update] to
+  the Bitcoin-Dev mailing list with summary of recent changes to the
```
Comment (adamjonas) [inline _posts/en/newsletters/2019-10-16-newsletter.md]: summary -> a summary

File: _posts/en/newsletters/2019-10-16-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+  and [transcripts][css ts] for several of the talks have been provided
+  by Bryan Bishop and others.  Out of the subset of talks that were
+  [transcribed][css ts], we thought the following three topics might be
+  of particular technical interest to the readers of this newsletter:
+
+    - *Everything is broken* by Cory Fields ([transcript][fields ts], [video][fields vid]), a
+      talk that describes how Bitcoin is at risk not just from its own
+      software bugs but also from the bugs introduced into the
+      libraries, operating systems, and even hardware that it depends
+      upon.  Fields then looks back in time when a profligation of
```
Comment (adamjonas) [inline _posts/en/newsletters/2019-10-16-newsletter.md]: While profligate is a great word, I don't think this use of profligation is quite correct. I had a hard time finding evidence of profligation used in this way elsewhere. Happy to be corrected.

File: _posts/en/newsletters/2019-10-16-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+      secure, compact, practical, and flexible scheme for
+      threshold-based schnorr signatures.  Ruffing first describes the
+      difference between generalized threshold signatures and the
+      specific case of multi-signatures.  A threshold signature allows a
+      subset of a group to sign (e.g. k-of-n); multi-signatures are a
+      special case of threshold signatures where the whole group signs
+      (n-of-n).  Protocols like MuSig (see [Newsletter #35][musig
+      libsecp256k1-zkp]) and [MSDL][] provide multi-signature signing
+      compatible with [bip-schnorr][].  Threshold signatures for a
+      subset of signers is not something that has been solved to the
```
Comment (adamjonas) [inline _posts/en/newsletters/2019-10-16-newsletter.md]: is -> are

**Comment by moneyball** [comment]
ACK

**Comment by bitschmidty** [comment]
ACK 
(pushed a small casing change to Coldcard and squashed)

Apologies for the late review!

**Comment by bitschmidty** [comment]
Software from LND, C-Lightning, and Eclair
Enough to make @harding lose his hair!

Please make sure nothings out of whack
Review the taproot BIPs and provide feedback

Everything is broken, lots couldve gone wrong
Despite all of that, Bitcoins still going strong

As always, TY to @harding for authoring and the last minute updates and to @jnewbery @adamjonas and @moneyball for the reviews and feedback

**Pull request merged**